rudder-core/src/main/scala/com/normation/rudder/services/quicksearch/QuickSearchBackendImpl.scala

```diff
@@ -276,6 +276,12 @@ object QSLdapBackend {
    * Mapping between attributes and their filter in the backend
    */
   object QSAttributeLdapFilter {
+
+    //the list of value to avoid in OC for matching
+    val ocRudderTypes = Set("top", OC_RUDDER_NODE, OC_POLICY_SERVER_NODE, OC_NODE
+                            , OC_RULE, OC_PARAMETER, OC_RUDDER_NODE_GROUP)
```

Comment:
shouldn't you avoid also OC_SPECIAL_TARGET, OC_ACTIVE_TECHNIQUE_LIB_VERSION, OC_ABSTRACT_RULE_WITH_CF3POLICYDRAFT, OC_RULE_WITH_CF3POLICYDRAFT, OC_RULE_WITH_CF3POLICYDRAFT, OC_NODE_CONFIGURATION,OC_PROPERTY, or are they filtered differently ?